shuup/simple_cms/admin_module/views.py

```diff
@@ -38,8 +47,10 @@ class Meta:
             'identifier',
             'visible_in_menu',
             'parent',
+            'template_path',
```

Comment:
replace `template_path` by `tamplate_name` a common used variable name to set templates